driver-sync/src/test/functional/com/mongodb/client/AbstractUnifiedTest.java

```diff
@@ -168,7 +169,16 @@ public void setUp() {
         collectionHelper.killAllSessions();
 
         if (!isDataLakeTest()) {
-            collectionHelper.create(collectionName, new CreateCollectionOptions(), WriteConcern.MAJORITY);
+            try {
+                collectionHelper.create(collectionName, new CreateCollectionOptions(), WriteConcern.MAJORITY);
+            } catch (MongoCommandException e) {
+                // Older sharded clusters sometimes reply with this error.  Work around it by retrying once.
+                if (e.getErrorCode() == 11601 && isSharded() && serverVersionLessThan(4, 2)) {
```

Comment:
I decided to just do the retry in the unified runner because it's a unified test that is failing.  But could also push the retry down into the helper method, which is used by other runners.

Reply:
If the bug is making other runners flaky, it seems the fix should be applied widely if possible.

I did not understand how to interpret the frequency of this bug in the chart - is retrying once enough to never practically see this bug again, or should this be a for loop?

Reply:
If the bug is making other runners flaky, it seems the fix should be applied widely if possible.

I did not understand how to interpret the frequency of this bug in the chart - is retrying once enough to never practically see this bug again, or should this be a for loop?

Reply:
For reasons I do not entirely understand, it's one specific unified test that is failing, and retrying once seems to do the trick in the multiple runs I've executed.  I don't want to retry until success as that could potentially cause the whole test suite to time out.

Reply:
I see, that makes sense